bsp/esp32_s3_korvo_1/idf_component.yml

```
@@ -15,3 +15,7 @@ dependencies:
   button:
     version: ">=2.5,<4.0"
     public: true
+
+  esp_codec_dev:
+    version: "1.0.3"
```

Let's allow automatic bugfix updates
```suggestion
    version: "~1.0.3"
```